hack/common.sh

```diff
@@ -35,11 +35,19 @@ OSDK_WMCO_management() {
   local COMMAND=$1
   local OSDK_PATH=$2
 
-  # Currently this fails even on successes, adding this check to ignore the failure
-  # https://github.com/operator-framework/operator-sdk/issues/2938
-  if ! $OSDK_PATH $COMMAND packagemanifests --olm-namespace openshift-operator-lifecycle-manager --operator-namespace openshift-windows-machine-config-operator \
-  --operator-version 0.0.0 ; then
-    echo operator-sdk $1 failed
+  local retries=5
+  for ((i=0; i<retries; i++)); do
+    $OSDK_PATH $COMMAND packagemanifests \
+      --olm-namespace openshift-operator-lifecycle-manager \
+      --operator-namespace openshift-windows-machine-config-operator \
+      --operator-version 0.0.0 \
```

Comment:
I don't think you can rerun `run packagemanifests` because I suspect it's not really failing in the first try. So I think it would be safer to do a clean between runs. Sorry I should have caught it on first review.

Reply:
That makes sense. I've opened #195 for this, and cherry-picked that to #193 and #194.